site/zenodo_rdm/files.py

```diff
@@ -55,6 +57,7 @@ def _get_eos_redirect_path(self):
     def send_file(self, filename, **kwargs):
```

Comment:
```suggestion
    def send_file(self, filename, as_attachment=False, **kwargs):
```

minor: can't we expose like this actually instead of cherry-picking `kwargs`? Actually I would expose all params from [the original `send_file`](https://github.com/inveniosoftware/invenio-files-rest/blob/74d89cc6cefdab267eacbe8bb0f8f37c2c715590/invenio_files_rest/storage/base.py#L103-L112), so that we don't forget to handle some of them.

Reply:
I assumed there was a reason of not implementing them initially, but good point, I will change it, thanks